nextflow.config

```diff
@@ -49,18 +49,12 @@ process {
     errorStrategy = { task.exitStatus in [130,137..140] ? 'retry' : 'terminate' }
     maxRetries = 5
 
-    cpus = { 1 * task.attempt * task.attempt}
-    memory = { 10.GB * task.attempt * task.attempt}
-
-}
-
-
-// see https://www.nextflow.io/docs/latest/config.html#scope-aws
-aws {
-  accessKey = '[YOUR-ACCESS-KEY]'
-  secretKey = '[YOUR-SECRET-KEY]'
-  client {
-        endpoint = 'https://cog.sanger.ac.uk'
-        signerOverride = "S3SignerType"
-  }
+    withLabel: small {
+        cpus = 1
+        memory = { 2.GB * task.attempt }
+    }
+    withLabel: large {
+        cpus = { 4 * task.attempt }
```

Comment:
Is the cpu number supposed to increase on each attempt?

Reply:
for the `image-to-zarr`. Yes. bf2raw can use as many cores as possible. For a large image. I would want it go faster with more core. But for the other process `Generate_image`. It's not really true.

Reply:
actually 20Gb might be too much as a starting point. Most laptops have <16Gb. I may just start with 10Gb memory.

Reply:
I don't think people using laptops will have an issue with that because they will run it using 'local executor' 🤔

Reply:
Hmm... In the current version, I think it applies to all users since no profile-specific allocation is in-place.
Or you mean it will omit this label when it's local?

Reply:
I'm pretty sure the local executor won't use those and will only honour the config for `executor{ memory=..., cpu=...}`
so they should be fine to keep as-is... I can double check latter with a tiny example if you want to.

Reply:
would be great. Cause it is reading it with local on my VM. Here's my dummy script
```
#!/usr/bin/env/ nextflow
// Copyright © 2023 Tong LI <tongli.bioinfo@proton.me>

nextflow.enable.dsl=2


process test {

    label 'huge'

    script:
    """
    echo 'hello world'
    """
}


workflow {
    test()
}

❯ more nextflow.config
process {
    withLabel: 'huge' {
	memory = 500.Gb
    }

Process requirement exceeds available memory -- req: 500 GB; avail: 336.4 GB
```

Reply:
I think the (dynamic) resource allocation has to be optional. The numbers are very dependant on the environment being used, and users might be frustrated that they need to iterate through cycles they know will fail when they already know what needs to be allocated.